py/server/deephaven/barrage.py

```diff
@@ -81,24 +145,27 @@ def barrage_session(host: str,
         try:
             return _get_barrage_session_uri(j_client_config, auth)
         except:
-            # fall back to the direct way when we don't have a fully initialized server, used for testing
+            # fall back to the direct way when we don't have a fully initialized server, used for testing only
+            # TODO: remove when we are done with restructuring the integrations tests wiring (a ticket to be filed by Colin Alworth.
```

Comment:
```suggestion
            # TODO: remove when we are done with restructuring the integration tests wiring (a ticket to be filed).
```
1. I think we've avoided names in the code.
2. Do we have a ticket number yte?